test/unstable-keyword/outermod/unstable_outermodule.set.good

```diff
@@ -0,0 +1,5 @@
+unstable_outermodule.chpl:11: In function 'main':
+unstable_outermodule.chpl:13: warning: Outer is unstable
+unstable_outermodule.chpl:22: warning: Outer is unstable
```

Comment:
Oho, an actual bug!  This test is missing a warning for line 20.  Which is interesting, because the warning on line 13 is still present.  I think this means that there was a case missed in either scopeResolve.cpp or ResolveScope.cpp